docs/flashbots-mev-boost/troubleshooting.md

```diff
@@ -0,0 +1,26 @@
+## Troubleshooting
```

Comment:
for troubleshooting i'd prefer to link to https://github.com/flashbots/mev-boost/wiki/Troubleshooting because it's easier to update

Reply:
maybe in this page link to the wiki?

for sure we can't have 2 places with troubleshooting docs. and i think ease of updating is a big plus